packages/token-bridge-sdk/src/withdrawals/__test__/fetchETHWithdrawalsFromEventLogs.test.ts

```diff
@@ -18,69 +18,41 @@ describe('fetchETHWithdrawalsFromEventLogs', () => {
 
   it('fetches some ETH withdrawals from event logs pre-nitro', async () => {
     const result = await fetchETHWithdrawalsFromEventLogs({
-      address: '0x41C966f99De0cA6F6531fbcAc9Db7eaBDF119744',
-      fromBlock: 11110000,
-      toBlock: 12055296,
+      address: '0xd898275e8b9428429155752f89fe0899ce232830',
+      fromBlock: 20785772,
+      toBlock: 22964111,
       l2Provider
     })
 
-    expect(result).toHaveLength(5)
+    expect(result).toHaveLength(1)
     expect(result).toEqual(
       expect.arrayContaining([
         expect.objectContaining({
           transactionHash:
-            '0xdc8f593a5c19f2133e59c6e332dee0f330d977ffa11387eee2cfc55cee1722e7'
-        }),
-        expect.objectContaining({
-          transactionHash:
-            '0xb482555a63edc154035834a7380ae3787c0126730228d4e1dce97520f25ed0f7'
-        }),
-        expect.objectContaining({
-          transactionHash:
-            '0xa639f70688a095ce3134c4768de5ef0e2fbad804ca4bdee8dc56131b4500bcb9'
-        }),
-        expect.objectContaining({
-          transactionHash:
-            '0xc3636c940f800ccc14e83a35448cc62ab9c5bd9bcdb25cc451769df9bd0260e0'
-        }),
-        expect.objectContaining({
-          transactionHash:
-            '0x821b91067d86d4081639183e29d00f1e553370f979514337f1aa2b867250a269'
+            '0x7378773d1af4cfbbc91179efdaf63872f8e1cb7f84e9a9511ef3f1ce6dbcb671'
         })
       ])
     )
   })
 
   it('fetches some ETH withdrawals from event logs pre-nitro and post-nitro', async () => {
     const result = await fetchETHWithdrawalsFromEventLogs({
-      address: '0x41C966f99De0cA6F6531fbcAc9Db7eaBDF119744',
-      fromBlock: 13744993,
-      toBlock: 13956985,
+      address: '0xd898275e8b9428429155752f89fe0899ce232830',
+      fromBlock: 22964112,
+      toBlock: 24905369,
```

Comment:
We should stretch this range a bit more into the past, so it also captures the withdrawals from the test above. We can set `fromBlock` to be `20785772`.

Reply:
Done

Reply:
NIT: Can we extract it to a constant to enforce it?